pom.xml

```diff
@@ -315,24 +349,36 @@
             <properties>
                 <spark.test.version>${spark301.version}</spark.test.version>
             </properties>
+            <modules>
+                <module>tests</module>
```

Comment:
`tests` is already a module that's included unconditionally?

Reply:
thanks for catching it, I moved to under some profiles and then moved it back and didn't remove these